src/components/navbar/Connected/Address.tsx

```diff
@@ -0,0 +1,21 @@
+import { AccentText } from 'components/Text'
+import { displayFromSm, displayFromXsToSm } from 'helpers/visibilityClassnames'
+import ENSAddress from 'components/ENSAddress'
+import Network from 'models/Network'
+
+export default function ({ account }: { account: string }) {
+  return (
+    <>
+      <span className={displayFromXsToSm}>
```

Comment:
We're `wet` again, let's `dry` once more

Reply:
Again, only one prop, let's not make it complicated

Reply:
Changed